docs/src/test/scala/docs/http/scaladsl/ServerSentEventsExampleSpec.scala

```diff
@@ -0,0 +1,44 @@
+package docs.http.scaladsl
+
+import akka.NotUsed
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.sse.ServerSentEvent
+import akka.http.scaladsl.server.{ Directives, Route, RoutingSpec }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.scaladsl.Source
+import docs.CompileOnlySpec
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter.ISO_LOCAL_TIME
```

Comment:
No, I don't think so. The important import for this use case is shown, the other imports (not just for Java Time) are "boiler plate".